OpenSim/Simulation/Wrap/WrapObject.cpp

```diff
@@ -178,3 +178,94 @@ int WrapObject::wrapPathSegment(const SimTK::State& s,
 
    return return_code;
 }
+
+void WrapObject::updateFromXMLNode(SimTK::Xml::Element& node,
+        int versionNumber) {
+    int documentVersion = versionNumber;
+    if (documentVersion < XMLDocument::getLatestVersion()) {
+        if (documentVersion < 30515) {
+            // Replace 3.3 display_preference, color, and VisibleObject with
+            // Appearance's visible and color.
+            // We ignore most of VisibleObject's other properties (e.g.,
+            // transform), since it would be misleading to draw the wrap object
+            // in the wrong place.
+            SimTK::Xml::Element appearanceNode("Appearance"); 
+            // Use the correct defaults for WrapObject's appearance, which
+            // are different from the default Appearance.
+            SimTK::Xml::Element color("color");
+            color.setValue("0 1 1"); // SimTK::Cyan
+            appearanceNode.insertNodeAfter(appearanceNode.element_end(),
+                    color);
+            SimTK::Xml::Element defaultOpacity("opacity");
+            defaultOpacity.setValue("0.5");
```

Comment:
Not sure about this, but would it make sense to set these values by creating a default WrapObject and looking up the values of its properties (in case the default property values change in the future)?